doc/gitian-building.md

```diff
@@ -305,6 +305,8 @@ Clone the git repositories for bitcoin and Gitian.
 ```bash
 git clone https://github.com/devrandom/gitian-builder.git
 git clone https://github.com/bitcoin/bitcoin
+git clone https://github.com/bitcoin/bitcoin-detached-sigs.git
```

Comment:
Is there a reason to clone bitcoin-detached-sigs manually? For me at least gbuild handles that for me